### ISSUE TITLE
Lab 14 Module 14 Terraform Demo fails with insufficient features blocks

### ISSUE THREAD

**Issue by christiannagel** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#99]
# Module: 14
#### Step: 6 - invalid terraform file:

The file shown to copy and paste does not work with the terraform version installed with the Azure Cloud Shell and returns the error **insufficient features blocks**.

```
provider "azurerm" {
}
resource "azurerm_resource_group" "rg" {
        name = "testResourceGroup"
        location = "westus"
}
```

#### Step 10 (`terraform apply`) returns with this error:

```
│ Error: Insufficient features blocks
│
│   on  line 0:
│   (source code not available)
│
│ At least 1 "features" blocks are required.
```

Adding an empty `features` block fixes the issue:

```
provider "azurerm" {
  features { }
}
resource "azurerm_resource_group" "rg" {
        name = "testResourceGroup"
        location = "westus"
}
```

**Comment by jphellemons** [comment]
Thank you for providing the fix! I have this exact issue at the moment during training.

**Comment by LuizMacedo** [comment]
Resolved by #118.
